File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/specialization/Specialization.java
package com.api.hospitalsystem.model.specialization;

public enum Specialization {

    GeneralPratictitioner("Clinico Geral"),
    Ginecology("Ginecologista"),
    Cardiologist("Cardiologista"),
    Surgeon("Cirurgião"),
    Dermatologist("Dermatologista"),
    Endocrinologist("Endocrinologista"),
    Geriatric("Geriatra"),
    Pediatrician("Pediatra"),
    Othorpedist("Ortopedista");

    private final String description;

    Specialization(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}

File: app/hospital-system/src/app/module/models/src/specialization/specialization.ts
import { describe } from '@module/utils/functions/enum';

export enum Specialization {
  GeneralPratictitioner = 0,
  Ginecology = 1,
  Cardiologist = 2,
  Surgeon = 3,
  Dermatologist = 4,
  Endocrinologist = 5,
  Geriatric = 6,
  Pediatrician = 7,
  Othorpedist = 8,
}  

describe(Specialization, {
  GeneralPratictitioner: 'Clinico Geral',
  Ginecology: 'Ginecologista',
  Cardiologist: 'Cardiologista',
  Surgeon: 'Cirurgião',
  Dermatologist: 'Dermatologista',
  Endocrinologist: 'Endocrinologista',
  Geriatric: 'Geriatra',
  Pediatrician: 'Pediatra',
  Othorpedist: 'Ortopedista',
});
